provisioner/provisioner.go

```diff
@@ -11,21 +11,30 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+type (
+	// A provider ID is a string that identifies a cluster provider.
+	ProviderID string
+
+	// Options is the options that can be passed to a provisioner when initialized.
+	Options struct {
+		DryRun          bool
+		ApplyOnly       bool
+		UpdateStrategy  config.UpdateStrategy
+		RemoveVolumes   bool
+		ManageEtcdStack bool
+	}
+)
+
+const (
+	ZalandoAWS ProviderID = "zalando-aws"
```

Comment:
nit: Maybe name the variable `ZalandoAWSProvider` to make it more clear what it is?

Reply:
Renamed.